homeassistant/components/keba/lock.py

```diff
@@ -0,0 +1,89 @@
+"""Support for KEBA charging station switch."""
+import logging
+
+from homeassistant.components.lock import LockDevice
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config,
+                               async_add_entities, discovery_info=None):
+    """Set up the KEBA charging station platform."""
+    if discovery_info is None:
+        return
+
+    keba = hass.data[DOMAIN]
+
+    sensors = [
+        KebaLock('Authentication', keba)
+    ]
+    async_add_entities(sensors)
+
+
+class KebaLock(LockDevice):
+    """The entity class for KEBA charging stations switch."""
+
+    def __init__(self, name, keba):
+        """Initialize the KEBA switch."""
+        self._keba = keba
+        self._name = name
+        self._state = None
+        self._attributes = {
+            'rfid_tag': self._keba.rfid
+        }
+
+    def open(self, **kwargs):
```

Comment:
If we don't do anything here, we can remove the method.

Reply:
ok, pyCharm told me I should implement all abstract methods, but sure I will delete it